src/react/commands/Eth2Deposit.ts

```diff
@@ -34,50 +40,53 @@ const requireDepositPackages = (): boolean => {
   }
 }
 
-const singleFileExecutableExists = (): boolean => {
-  access(SFE_PATH, constants.F_OK, (err) => {
-    return false;
-  });
-  return true;
-}
-
-const createMnemonic = (language: string): string => {
-  let cmd = "";
-  let env = process.env;
-
-  const escapedLanguage = escapeArgument(language);
-
-  if(singleFileExecutableExists()) {
-    cmd = SFE_PATH + " " + CREATE_MNEMONIC_SUBCOMMAND + " " + DIST_WORD_LIST_PATH + " --language " + escapedLanguage;
-    console.log('Calling SFE for create mnemonic');
-  } else {
-    if(!requireDepositPackages()) {
-      return '';
-    }
-  
-    const pythonpath = executeCommandSync("python3 -c \"import sys;print(':'.join(sys.path))\"");
-  
-    const expythonpath = REQUIREMENT_PACKAGES_PATH + ":" + ETH2_DEPOSIT_CLI_PATH + ":" + pythonpath;
-  
-    env.PYTHONPATH = expythonpath;
+const createMnemonic = (language: string): Promise<string> => {
+  return new Promise((resolve, reject) => {
```

Comment:
I'm guessing there was a good reason this make `createMnemonic` return a Promise. For UI maybe? Shouldn't we do the same for `generateKeys`?

Reply:
Yes, just created #61 for it.  I was able to get around reimplementing it for now, but I do think generateKeys should be rewritten the same way.